lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -820,6 +820,17 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             course_grade.letter_grade,
             student.id in whitelisted_user_ids
         )
+        if certificate_info[0] == 'Y':
+            TASK_LOG.info(u'Student is marked eligible_for_certificate in grade report (user=%s, course_id=%s,'
```

Comment:
@asadazam93 , check out our guidelines for breaking long lines: edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines

Also, let's only log when `certificate_info[0] == 'Y'` AND when the student's grade in the course isn't a passing grade (it's below the minimum grade in the course) AND when they're not whitelisted?